tutorials/local/MT/df001_introduction.py

```diff
@@ -0,0 +1,111 @@
+## \file
+## \ingroup tutorial_dataframe
+## \notebook -nodraw
+## This tutorial illustrates the basic features of the RDataFrame class,
+## a utility which allows to interact with data stored in TTrees following
+## a functional-chain like approach.
+##
+## \macro_code
+##
+## \date May 2017
+## \author Danilo Piparo
+
+import ROOT, PyRDF
+
+# A simple helper function to fill a test tree: this makes the example stand-alone.
+def fill_tree(treeName, fileName):
+    tdf = ROOT.ROOT.RDataFrame(10)
+    tdf.Define("b1", "(double) tdfentry_")\
+       .Define("b2", "(int) tdfentry_ * tdfentry_").Snapshot(treeName, fileName)
+
+# We prepare an input tree to run on
+fileName = "df001_introduction_py.root"
+treeName = "myTree"
+fill_tree(treeName, fileName)
+
+
+# We read the tree from the file and create a RDataFrame, a class that
+# allows us to interact with the data contained in the tree.
+RDF = PyRDF.RDataFrame
+d = RDF(treeName, fileName)
+ROOT.ROOT.EnableImplicitMT()
```

Comment:
If you enable it here, it won't run multi-threaded because the local backend has already been constructed, right?
You should start all the notebooks with
```
import ROOT
ROOT.ROOT.EnableImplicitMT()
import PyRDF
```

Reply:
Ok @etejedor 👍

Reply:
@etejedor , I've made that change now anyway 👍

Reply:
Right, it will run MT but we still have to enable it before constructing the local backend, just in case there is a Range that we need to detect.